package/MDAnalysis/coordinates/DCD.py

```diff
@@ -391,15 +401,15 @@ class DCDReader(base.ReaderBase):
         ``data = dcd.correl(...)``
            populate a :class:`MDAnalysis.core.Timeseries.Collection` object with computed timeseries
 
-    .. Note::
+    Note
+    ----
 
```

Comment:
Done (empty line not harmful and positively helps when reformatting the block in emacs but I agree that this should be consistent everywhere)